### PR TITLE
Increase max length, add description to task

### PR DESCRIPTION
**Pull request by shirishgoyal**

### CODE REVIEW

File: crowdsourcing/models.py [hunk 1]
```diff
@@ -208,7 +208,7 @@ class Module(models.Model):
             -repetition: number of times a task needs to be performed
     """
     name = models.CharField(max_length=128, error_messages={'required': "Please enter the module name!"})
-    description = models.TextField(error_messages={'required': "Please enter the module description!"})
+    description = models.TextField(error_messages={'required': "Please enter the module description!"}, max_length=2048)
```
Comment (dmorina) [inline crowdsourcing/models.py]: default VARCHAR is much bigger that this no need to limit it

**Comment by dmorina** [comment]
lgtm

File: crowdsourcing/models.py (hunk 1 above)
Comment (shirishgoyal) [inline crowdsourcing/models.py]: It is always better to explicitly mention the length, makes debugging easier.

**Comment by neilthemathguy** [comment]
lgtm

**Pull request merged**